public/locales/de/donate.ts

```diff
@@ -0,0 +1,54 @@
+import { ILocaleObject, ILocaleParams } from './../localeInterfaces';
+
+const donate: ILocaleObject = {
+  topProjects: 'Top Projekte',
+  allCountProjects: ({ projectCount }: ILocaleParams) =>
```

Comment:
I do not know if LingoHub can work with TypeScript files. (I doubt it.) Especially when writing functions within the resource files to fill placeholders. Let's hope I am wrong. Otherwise how should we give this to translators?

Reply:
Actually you are right. Just had a talk with Sagar also checked the format LingoHub supports this won't work.

Reply:
I demonstrated how to use multiple JSON files and placeholders in my pull request https://github.com/Plant-for-the-Planet/planet-webapp/pull/236.

Reply:
@ankitecd would you like to move your translation to #236 ?